src/seer/automation/codebase/repo_client.py

```diff
@@ -102,6 +105,7 @@ class RepoClientType(str, Enum):
     READ = "read"
     WRITE = "write"
     CODECOV_UNIT_TEST = "codecov_unit_test"
+    CODECOV_PR_REVIEW = "codecov_pr_review"
```

Comment:
Hmm would it be simpler to just combine these enum types into one?
```suggestion
    CODECOV = "codecov"
```

Reply:
I think I will come back to this once our services are able to talk to each other and update accordingly along with the app id. For now, I prefer to separate them out.